Dockerfile

```
@@ -1,37 +1,39 @@
+#   (Stop it with "docker-compose down prod")
+#
+# Or just use pnpm scripts from package.json
 # -----------------------------------------------------------------------------
 
-# Base stage - npm installation
+# Base stage - pnpm installation
 FROM node:24-alpine@sha256:b13895216ed51cc6decab51466f86346884af1e3656dea02a2a9c1a7c4dedccd AS base
 WORKDIR /app
-COPY package*.json ./
-RUN npm ci
+RUN npm install -g pnpm
```

Hadolint (2.12.0)

[warning] Pin versions in npm. Instead of `npm install <package>` use `npm install <package>@<version>`

(DL3016)